src/main/java/com/minecrafttas/tasmod/handlers/KeybindingHandler.java

```diff
@@ -0,0 +1,86 @@
+package com.minecrafttas.tasmod.handlers;
+
+import org.lwjgl.input.Keyboard;
+
+import com.minecrafttas.common.events.client.EventClientGameLoop;
+import com.minecrafttas.tasmod.TASmod;
+import com.minecrafttas.tasmod.TASmodClient;
+import com.minecrafttas.tasmod.externalGui.InputContainerView;
+import com.minecrafttas.tasmod.playback.PlaybackController.TASstate;
+import com.minecrafttas.tasmod.playback.server.TASstateClient;
+import com.minecrafttas.tasmod.savestates.server.LoadstatePacket;
+import com.minecrafttas.tasmod.savestates.server.SavestatePacket;
+import com.minecrafttas.tasmod.virtual.VirtualKeybindings;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.settings.KeyBinding;
+
+/**
+ * 
+ * The class where all keybinding functionality is executed
+ * @author Scribble
+ *
+ */
+public class KeybindingHandler implements EventClientGameLoop {
+
+	public KeyBinding tickratezeroKey = new KeyBinding("Tickrate 0 Key", Keyboard.KEY_F8, "TASmod");
+
+	public KeyBinding tickAdvance = new KeyBinding("Advance Tick", Keyboard.KEY_F9, "TASmod");
+
+	public KeyBinding stopkey = new KeyBinding("Recording/Playback Stop", Keyboard.KEY_F10, "TASmod");
+
+	public KeyBinding savestateSaveKey = new KeyBinding("Create Savestate", Keyboard.KEY_J, "TASmod");
+
+	public KeyBinding savestateLoadKey = new KeyBinding("Load Latest Savestate", Keyboard.KEY_K, "TASmod");
+
+	public KeyBinding testingKey = new KeyBinding("Various Testing", Keyboard.KEY_F12, "TASmod");
+
+	public KeyBinding infoGuiKey = new KeyBinding("Open InfoGui Editor", Keyboard.KEY_F6, "TASmod");
+	
+	public KeyBinding bufferViewKey = new KeyBinding("Buffer View", Keyboard.KEY_NUMPAD0, "TASmod");
```

Comment:
why the newlines

Reply:
Because I can never make this code look good
![eclipse_KUB0WI5vVc](https://github.com/MinecraftTAS/TASmod/assets/28013575/1ecea58c-b4c2-450c-aa9f-b85c35378093)